src/items.zig

```diff
@@ -353,11 +345,24 @@ const ToolPhysics = struct { // MARK: ToolPhysics
 		}
 		var tempModifiers: main.List(Modifier) = .init(main.stackAllocator);
 		defer tempModifiers.deinit();
+		for(tool.type.properties) |property| {
+			var sum: f32 = 0;
+			var weight: f32 = 0;
+			for(0..25) |i| {
+				const material = (tool.craftingGrid[i] orelse continue).material orelse continue;
+				sum += property.weigths[i]*material.getProperty(property.source);
+				weight += property.weigths[i];
+			}
+			if(weight == 0) continue;
+			sum /= weight;
+			sum *= property.resultScale;
+			tool.getProperty(property.destination).* += sum;
+		}
+		if(tool.maxDurability < 1) tool.maxDurability = 1;
+		if(tool.damage < 1) tool.damage = 1/(2 - tool.damage);
```

Comment:
What/Why is that? What will be the result for -inf?

Reply:
I want to allow for negative input values without making the result negative, which would have unintended effects for damage (healing) and time to swing (instant breaking is too easy to get)

Here is a plot, it approaches 0 as x→-∞
![Screenshot at 2025-04-22 20-25-24](https://github.com/user-attachments/assets/5efa4d1a-25bb-40de-b43b-afe6ab59734e)

Reply:
Alright, its nice function.